nautobot/docs/installation/selinux-troubleshootiing.md

```diff
@@ -0,0 +1,199 @@
+# SELinux Troubleshooting
+
+When installing Nautobot for the first time on a Redhat-based Linux Distribution, SELinux may prevent the Nautobot stack from working properly. An example is SELinux prevents the HTTP daemon (Nginx, Apache, etc) from communicating to the Django application stack on the backend.
+
+## Determine if SELinux is the Culprit
+
+An example a broken application can be seen in the Nginx error logs below:
+
+```no-highlight
+$ sudo tail -f /var/log/nginx/error.log
+2021/02/26 15:16:55 [crit] 67245#0: *494 connect() to 127.0.0.1:8001 failed (13: Permission denied) while connecting to upstream, client: 47.221.167.40, server: nautobot.example.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8001/", host: "nautobot.example.com"
+2021/02/26 15:16:56 [crit] 67245#0: *494 connect() to 127.0.0.1:8001 failed (13: Permission denied) while connecting to upstream, client: 47.221.167.40, server: nautobot.example.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8001/favicon.ico", host: "nautobot.example.com", referrer: "https://nautobot.example.com/"
+2021/02/26 15:16:58 [crit] 67245#0: *544 connect() to 127.0.0.1:8001 failed (13: Permission denied) while connecting to upstream, client: 47.221.167.40, server: nautobot.example.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8001/", host: "nautobot.example.com"
+2021/02/26 15:16:58 [crit] 67245#0: *544 connect() to 127.0.0.1:8001 failed (13: Permission denied) while connecting to upstream, client: 47.221.167.40, server: nautobot.example.com, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8001/favicon.ico", host: "nautobot.example.com", referrer: "https://nautobot.example.com/"
+```
+
+A quick way to verify that SELinux is preventing the application from working is to first verify that SELinux is indeed `enabled` and operating in `enforcing` mode and second; temporarily put SELinux in `permissive` mode. With SELinux in `permissive` mode, the application stack can be tested again. If the application starts working as expected, then SELinux is most likely the culprit.
+
+```no-highlight
+# sestatus | egrep 'SELinux status|Current mode'
+SELinux status:                 enabled
+Current mode:                   enforcing
+```
+
+To put SELinux in `permissive` mode, execute the `setenforce` command with the `0` flag.
+
+```no-highlight
+# sestatus | egrep 'SELinux status|Current mode'
+SELinux status:                 enabled
+Current mode:                   enforcing
```

Comment:
Nitpick: Not sure if these three lines are needed since they're just repeating what's immediately above.